akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -69,6 +66,9 @@ object Flow {
     sink: Graph[SinkShape[I], M1], source: Graph[SourceShape[O], M2],
     combine: function.Function2[M1, M2, M]): Flow[I, O, M] =
     new Flow(scaladsl.Flow.fromSinkAndSourceMat(sink, source)(combinerToScala(combine)))
+
+  def monitorFlow[T](): Flow[T, T, FlowMonitor[T]] =
```

Comment:
I think it should have the same name in all three places. `monitor` is good in my opinion.